x/gov/README.md

```diff
@@ -116,27 +120,26 @@ proposal is finalized (passed or rejected).
 #### Deposit refund and burn
 
 When a proposal is finalized, the coins from the deposit are either refunded or burned
-according to the final tally of the proposal:
+according to the final tally of the proposal. In Heimdall, burn is not enabled, hence all the deposits will be refunded.
 
 * If the proposal is approved or rejected but *not* vetoed, each deposit will be
   automatically refunded to its respective depositor (transferred from the governance
   `ModuleAccount`).
-* When the proposal is vetoed with greater than 1/3, deposits will be burned from the
+* When the proposal is vetoed with greater than 1/3, deposits will be refunded from the
   governance `ModuleAccount` and the proposal information along with its deposit
   information will be removed from state.
-* All refunded or burned deposits are removed from the state. Events are issued when
-  burning or refunding a deposit.
+* All refunded deposits are removed from the state. Events are issued when
+  refunding a deposit.
 
 ### Vote
 
 #### Participants
 
 *Participants* are users that have the right to vote on proposals. On the
-Cosmos Hub, participants are bonded Atom holders. Unbonded Atom holders and
-other users do not get the right to participate in governance. However, they
-can submit and deposit on proposals.
+Polygon PoS network, participants are validators. Other holders and users do not get the right to participate in governance. 
+However, they can submit and deposit on proposals.
```

Comment:
Hmm I don't think non-validators can submit/deposit on proposals can they ?

Reply:
You would need to add an additional check then on gov proposal submission, the proposer would need to be in the active set

Reply:
Ok addressed this with a TODO, will clarify later what to do (based also on discussion with `gov` team